crates/sui/src/sui_move/build.rs

```diff
@@ -13,25 +14,40 @@ pub struct Build {
     /// Whether we are printing in base64.
     #[clap(long, global = true)]
     pub dump_bytecode_as_base64: bool,
+    /// If true, generate struct layout schemas for
+    /// 1. All types declared by modules in this package
+    /// 2. All struct types passed into `entry` functions declared by modules in this package
+    /// These layout schemas can be consumed by clients (e.g.,
+    /// the TypeScript SDK) to enable serialization/deserialization of transaction arguments
+    /// and events.
+    pub generate_struct_layouts: bool,
 }
 
 impl Build {
-    pub fn execute(&self, path: Option<PathBuf>, build_config: BuildConfig) -> anyhow::Result<()> {
+    pub fn execute(
+        &self,
+        path: Option<PathBuf>,
+        build_config: MoveBuildConfig,
+    ) -> anyhow::Result<()> {
         let rerooted_path = base::reroot_path(path)?;
         Self::execute_internal(&rerooted_path, build_config, self.dump_bytecode_as_base64)
     }
+
     pub fn execute_internal(
         rerooted_path: &Path,
-        build_config: BuildConfig,
+        config: MoveBuildConfig,
         dump_bytecode_as_base64: bool,
     ) -> anyhow::Result<()> {
-        // find manifest file directory from a given path or (if missing) from current dir
+        let pkg = sui_framework::build_move_package(
+            rerooted_path,
+            BuildConfig {
+                config,
+                run_bytecode_verifier: false,
+                print_diags_to_stderr: true,
+            },
```

Comment:
Maybe `BuildConfig { config, print_diags_to_stderr: true, ..BuildConfig::default() }`

Reply:
I want to be explicit here in case we end up changing the defaults for the verifier (it should always run here even if it's off for tests).